tizen/src/platform_tizen.cpp

```diff
@@ -220,9 +226,103 @@ std::string stringFromFile(const char* _path) {
     return out;
 }
 
-// No system fonts fallback implementation (yet!)
+void initPlatformFontSetup() {
```

Comment:
A simple local static `isInitialized` within `getFallback` should do the job. And on the first call to it branch to everything within `initPlatformFontSetup`.